src/icons/collections/IconCollectionsStar.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlNeutral03 } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    selected?: boolean,
```

Comment:
to andrew's point, this doesn't really belong on the Icon. `is-selected` should be passed in as a classname from the parent since these 'icons' are supposed to be dumb icon components that don't have a 'selected' state.

All the collections icons need to be changed to adhere to this